arunoda/learnnextjs-demo
# Replace branch approach by monorepo

- each lesson is a folder (each item in the next-site/learn sidebar is a folder, with the exception of getting started)
- update dependencies to latest
- add prettier to fix formatting conflicts
- it is not needed anymore to stop/start next
- add readme to the repository explaining how to clone/download the repo
- fix 'url' property is deprecated
- add name to unnamed components
- use functions instead of arrow functions

todo :
- [x] fix [this issue](https://github.com/zeit/next.js/issues/6240) in E2-firebase-hn
- [x] create PR with appropriate changes in next-site/learn

room for improvement:
- [ ] add cross-env for `NODE_ENV=production node server.js`
- [ ] replace `webpack-bundle-analyzer` by `@zeit/next-bundle-analyzer`
- [ ] lazy loading components lesson needs to be updated

fixes https://github.com/arunoda/learnnextjs-demo/issues/26
fixes https://github.com/arunoda/learnnextjs-demo/issues/24

@timneutkens 
Which one do you prefer:
- the content of a folder is what is expected at the end of the lesson (e.g. `1-getting-started` contains what you get when you're done with the *Getting Started* lesson)
- the content of a folder is what you should start with at the beginning of a lesson (e.g. `2-using-shared-components` contains the files to start the *Using shared components* lesson)
